sagemaker-clarify/time_series_byom/time_series_bring_your_own_model.ipynb

```diff
@@ -0,0 +1,1019 @@
+      "\u001b[?25hUsing cached s3transfer-0.10.1-py3-none-any.whl (82 kB)\n",
+      "\u001b[33mWARNING: Ignoring invalid distribution -otocore (/local/home/zicanl/.virtualenvs/venv/lib/python3.9/site-packages)\u001b[0m\u001b[33m\n",
+      "\u001b[0mInstalling collected packages: botocore, s3transfer, boto3, sagemaker\n",
+      "  Attempting uninstall: botocore\n",
+      "    Found existing installation: botocore 1.29.165\n",
+      "    Uninstalling botocore-1.29.165:\n",
+      "      Successfully uninstalled botocore-1.29.165\n",
+      "  Attempting uninstall: s3transfer\n",
+      "    Found existing installation: s3transfer 0.6.2\n",
+      "    Uninstalling s3transfer-0.6.2:\n",
+      "      Successfully uninstalled s3transfer-0.6.2\n",
+      "  Attempting uninstall: boto3\n",
+      "    Found existing installation: boto3 1.26.83\n",
+      "    Uninstalling boto3-1.26.83:\n",
+      "      Successfully uninstalled boto3-1.26.83\n",
+      "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
+      "awscli 1.31.7 requires botocore==1.33.7, but you have botocore 1.34.80 which is incompatible.\n",
+      "awscli 1.31.7 requires s3transfer<0.9.0,>=0.8.0, but you have s3transfer 0.10.1 which is incompatible.\n",
+      "mercury 2.3.7 requires boto3==1.26.83, but you have boto3 1.34.80 which is incompatible.\u001b[0m\u001b[31m\n",
+      "\u001b[0mSuccessfully installed boto3-1.34.80 botocore-1.34.80 s3transfer-0.10.1 sagemaker-2.214.3\n",
+      "\u001b[33mWARNING: Ignoring invalid distribution -otocore (/local/home/zicanl/.virtualenvs/venv/lib/python3.9/site-packages)\u001b[0m\u001b[33m\n",
+      "\u001b[0mName: sagemaker\n",
+      "Version: 2.214.3\n",
+      "Summary: Open source library for training and deploying models on Amazon SageMaker.\n",
+      "Home-page: https://github.com/aws/sagemaker-python-sdk/\n",
+      "Author: Amazon Web Services\n",
+      "Author-email: \n",
+      "License: Apache License 2.0\n",
+      "Location: /local/home/zicanl/.virtualenvs/venv/lib/python3.9/site-packages\n",
+      "Requires: attrs, boto3, cloudpickle, docker, google-pasta, importlib-metadata, jsonschema, numpy, packaging, pandas, pathos, platformdirs, protobuf, psutil, PyYAML, requests, schema, smdebug-rulesconfig, tblib, tqdm, urllib3\n",
+      "Required-by: \n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip uninstall sagemaker -y -q c\n",
+    "!pip install sagemaker\n",
+    "!pip show sagemaker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7597402",
+   "metadata": {},
+   "source": [
+    "### Import Libraries\n",
+    "\n",
+    "The model used in this example notebook is DeepAr forecasting model in AWS. For more information, please check https://docs.aws.amazon.com/sagemaker/latest/dg/deepar.html (TODO, change this)\n",
```

Comment:
This notebook is for the BYOM demo, correct? Description says this uses the DeepAR forecasting model. Please update.